org.civicrm.reqangular/src/common/services/pub-sub-service.js

```diff
@@ -0,0 +1,32 @@
+define([
+  'common/modules/services'
+], function (module) {
+  'use strict';
+
+  var topics = Object.create(null);
+  module.factory("pubSubService", ['$rootScope', function ($rootScope) {
```

Comment:
What about documenting this service? We don't do it yet, it would be good to start